### PR TITLE
Greeting with color

### PR DESCRIPTION
**Pull request by mcfreund**

### CODE REVIEW

**Comment by timholy** [comment]
This needs to be reviewed by the repository owner. EDIT: I notice this PR is recent, so not casting any blame here.

**Review by nlaird — APPROVED** [review]
Don't you think this would look better in :magenta?

File: src/DemoPackageNML.jl [hunk 1]
```diff
@@ -1,5 +1,5 @@
 module DemoPackageNML
 
-print_greeting() = print("Hello, world")
+print_greeting() = printstyled("Hello, world!"; color=:cyan)
```
Comment (nlaird) [inline src/DemoPackageNML.jl]: Don't you think this would look better in :magenta?

**Review by nlaird — APPROVED** [review]
Thanks for the pull request!

File: src/DemoPackageNML.jl (hunk 1 above)
Comment (timholy) [reply src/DemoPackageNML.jl]: bump

**Review by nlaird — COMMENTED** [review]
Thanks for the change!

**Pull request merged**